lib/kernel/test/interactive_shell_SUITE.erl

```diff
@@ -111,7 +113,28 @@ groups() ->
        remsh_no_epmd,
        remsh_expand_compatibility_25,
        remsh_expand_compatibility_later_version]},
-     {tty,[],
+     {ssh, [],
+        [{group,ssh_unicode},
+         {group,ssh_latin1},
+         shell_ignore_pager_commands
+        ]},
+       {ssh_unicode,[],
+        [{group,ssh_tests}]},
+       {ssh_latin1,[],[{group,ssh_tests}]},
+       {ssh_tests, [],
+        [shell_navigation, shell_multiline_navigation,
+         shell_multiline_prompt_ssh,
+         shell_xnfix, shell_delete,
+         shell_transpose, shell_search, shell_insert,
+         shell_update_window,
+         shell_small_window_multiline_navigation,
+         shell_huge_input,
+         shell_support_ansi_input,
+         shell_receive_standard_out,
+         shell_expand_location_above,
+         shell_expand_location_below,
+         shell_clear]},
```

Comment:
I wonder if perhaps we should move the tests that are common to both tty and shell to a separate function, or maybe a separate group? So that it is natural to add tests in both categories when adding new testcases.